src/pages/dataTableUtilities/actions.js

```diff
@@ -380,3 +382,11 @@ export const resetStocktake = () => (dispatch, getState) => {
 
   dispatch({ type: 'resetStocktake' });
 };
+
+export const applyReason = value => (dispatch, getState) => {
+  const { modalValue } = getState();
+
+  modalValue.applyReasonToBatches(UIDatabase, value);
```

Comment:
should rename `applyReasonToBatches` to `applyReason` so that it is a common api between `StocktakeItem` and `StocktakeBatch` - can then reuse this action creator